src/edu/washington/escience/myria/MyriaConstants.java

```diff
@@ -320,9 +325,89 @@ private MyriaConstants() {}
     QUERY
   };
 
-  /** Number of bytes per worker partition for parallel ingest - 100MB */
+  /**
+   * python executable name.
+   */
+  public static final String PYTHONEXEC = "python";
+  /**
+   * Python worker module.
+   */
+  public static final String PYTHONWORKER = "MyriaPythonWorker.worker";
+
+  /**
+   * UDF language.
+   */
+  public static enum FunctionLanguage {
+    /** Postgres UDF. */
+    POSTGRES,
+    /** Python UDF. */
+    PYTHON
+  };
+  /**
+   * Python UDF relation: relation for holding python functions on each worker.
+   */
+  public static final RelationKey PYUDF_RELATION = new RelationKey("public", "function", "Python");
+  /**
+   * Python SpecialLenghts: send/received to the python worker.
+   */
+  public static enum PythonSpecialLengths {
+    PYTHON_EXCEPTION(-3), /** python exception returned from python process.*/
+    NULL_LENGTH(-5), /** python function return is null.*/
+    EOS(-4); /** Send EOS to python worker.*/
```

Comment:
I can't make the start be zero, the Java-to-python communication protocol needs this to not be a positive integer.